### PR TITLE
Detect pass alert/v7

### PR DESCRIPTION
**Pull request by victorjulien**
#11260 rebased, with review comments addressed

SV_BRANCH=https://github.com/OISF/suricata-verify/pull/1903

https://redmine.openinfosecfoundation.org/issues/5466

### CODE REVIEW

**Comment by suricata-qa** [comment]
Information: QA ran without warnings. 

Pipeline 21007

File: src/detect-noalert.c [hunk 1]
```diff
@@ -38,11 +38,25 @@ static int DetectNoalertSetup(DetectEngineCtx *de_ctx, Signature *s, const char
...
 {
     sigmatch_table[DETECT_NOALERT].name = "noalert";
     sigmatch_table[DETECT_NOALERT].desc = "no alert will be generated by the rule";
     sigmatch_table[DETECT_NOALERT].url = "/rules/flow-keywords.html";
     sigmatch_table[DETECT_NOALERT].Setup = DetectNoalertSetup;
     sigmatch_table[DETECT_NOALERT].flags |= SIGMATCH_NOOPT;
+
+    sigmatch_table[DETECT_ALERT].name = "alert";
+    sigmatch_table[DETECT_ALERT].desc = "alert will be generated by the rule";
+    sigmatch_table[DETECT_ALERT].url = "/rules/flow-keywords.html";
```
Comment (inashivb) [inline src/detect-noalert.c]: seems like this typo fix got missed. Rest looks good. 🚀

Comment (inashivb) [reply src/detect-noalert.c]: oops. there's another commit later on.

**Review by inashivb — APPROVED** [review]
LGTM 🚀

**Comment by victorjulien** [comment]
Merged in #11275, thanks!

**Pull request merged**